main.ts

```diff
@@ -486,6 +487,7 @@ function addPlayerToDataSafe(data: Player, socket: Socket): void {
           socket
       );
       console.log('⚠️ invalid player data received');
+      console.log(error.message);
```

Comment:
probably comment this, the output is quite long and generally not needed when not debugging